### DISCUSSION TITLE
Precursor charge range

### DISCUSSION THREAD

**Discussion by fsun0403** [vdemichev/DiaNN#765] [Q&A]
Hello,

I would like know what precursor charge range should be set for DIA-NN. Tryptic peptides normally have more than one charge. When I used the default setting of precursor charge range 1-4, there were still lots of peptides with only one charge.  When charge 2-5 was used, the number of identified peptides decreased. However, some of the same peptides that appeared in the two separate search results were labeled with different charges (one or two charges).

**Comment by vdemichev** [comment]
Default is usually the best, restricting to 2-4 or even 2-3 is sometimes beneficial though. You can always test and see which charges get detected in your sample.
